Source/VaRest/Private/VaRestSubsystem.cpp

```diff
@@ -107,6 +108,73 @@ UVaRestJsonObject* UVaRestSubsystem::ConstructVaRestJsonObject()
 	return NewObject<UVaRestJsonObject>(this);
 }
 
+UVaRestJsonValue* UVaRestSubsystem::ConstructJsonValueNumber(float Number)
+{
+	TSharedPtr<FJsonValue> NewVal = MakeShareable(new FJsonValueNumber(Number));
+
+	UVaRestJsonValue* NewValue = NewObject<UVaRestJsonValue>();
```

Comment:
For my mind it's a good idea to set Outer here (and below) to `this`. What do you think about it?

Reply:
Done